examples/calibrateTask.py

```diff
@@ -117,15 +120,14 @@ def loadAndMatch(self, exposure, sourceCat):
 
 def run(display=False):
     #
-    # Create the task using a butler constructed using the obs_test repo
+    # Create the tasks
     #
-    butler = dafPersistence.Butler(os.path.join(lsst.utils.getPackageDir("obs_test"), "data", "input"))
```

Comment:
I'm pretty sure you can also delete the line that imports `dafPersistence`. (If so flake8 should show you that; please do run that).